go-controller/pkg/network-attach-def-controller/network_attach_def_controller.go

```diff
@@ -71,23 +71,28 @@ func NewNetAttachDefinitionController(
 	ncm NetworkControllerManager,
 	wf watchFactory,
 ) (*NetAttachDefinitionController, error) {
-	nadInformer := wf.NADInformer()
```

Comment:
should be in some other commit?

Reply:
this is changed to mock watchFactory for the test, so thinking it may not need another commit.